CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
 ### Bug fixes
 * Changed `BiologicalSequence.distance` to raise an error any time two sequences are passed of different lengths regardless of the `distance_fn` being passed. [(#514)](https://github.com/biocore/scikit-bio/issues/514)
 * Fixed issue with ``TreeNode.extend`` where if given the children of another ``TreeNode`` object (``tree.children``), both trees would be left in an incorrect and unpredictable state. ([#889](https://github.com/biocore/scikit-bio/issues/889))
+* Fixed issue where non-symmetric distance matrices were being created by `randdm`.
```

Comment:
This is minor but important: the current wording implies that `randdm` actually created asymmetric distance matrices. The function *tried* to create asymmetric distance matrices but an error was raised by the `DistanceMatrix` constructor when this happened. Can you reword so that this is clearer? The current wording makes it sound like invalid data was being created by `randdm`, which is far worse of a bug than having an error raised.